lib/winston/logger.js

```diff
@@ -118,14 +118,24 @@ Logger.prototype.extend = function (target) {
 //
 Logger.prototype.log = function (level) {
   var self = this,
-      args = Array.prototype.slice.call(arguments, 1);
+      timestamp,
+      args;
+
+  if (new Date(level).toString() !== 'Invalid Date') {
```

Comment:
This is too much of a performance hit. I've been working on a number of benchmarks for comparing `winston@2.x` to `winston@3.x` and this will be a big hit.